### DISCUSSION TITLE
Add NPC to Replay Arena Matches for Review and Integrity

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#7383] [Suggestions]
I'd like to propose the addition of an NPC feature that allows players to replay arena matches. This feature would provide players and the community with the ability to review past matches for personal improvement, entertainment, and, importantly, to ensure competitive integrity by detecting win-trading or scripting.

**Suggested Features:**

1. NPC Location:

    - Place the NPC near all the arena battlemasters for easy access by all players.

1. Match Accessibility:

   - Implement filters to search by player name, arena team name, match ID (that can be looked up on the website).

This feature could strengthen the integrity of competitive play while also offering more tools for players to improve and engage with the game on a deeper level. Thank you for considering this suggestion!

**Comment by Helias** [comment]
I think we can close this discussion ;-).

Thanks for your suggestion!

https://www.youtube.com/watch?v=P7hYa2rwWqo
https://www.chromiecraft.com/en/pvp-revamp-arena-s3/
